tests/wandb_integration_test.py

```diff
@@ -183,6 +183,7 @@ def test_network_fault_files(live_mock_server, test_settings):
     )
 
 
+@pytest.mark.timeout(120)
 def test_network_fault_graphql(live_mock_server, test_settings):
```

Comment:
Hmmm, I wonder why it's taking that long.  Is it the backoff?  We could probably decrease `fail_graphql_times`...

Reply:
Yeah, i just want to get these tests less flaky.  I don't want to defeature functionality as I am not sure what the 5 graphql requests are right now, (viewer query - maybe *2, then an upsert? * 3?)

We can take another pass making the unittests more efficiently use time.

Reply:
I added a ticket and a comment to look into this in the future:
https://wandb.atlassian.net/browse/WB-4365